backend/api/resource/users/model.go

```diff
@@ -21,6 +21,11 @@ type Form struct {
 	Password string `json:"password" form:"required,password,max=255"`
 }
 
+type UpdateForm struct {
+	Name  string `json:"name" form:"required,alpha_space,max=255"`
+	Email string `json:"email" form:"required,email,max=255"`
```

Comment:
I think all fields should be optional, but at least one of them must be filled in. I suggest creating a validation rule for this, [example](https://github.com/go-playground/validator/issues/195)